src/modules/GameConstants.ts

```diff
@@ -1579,6 +1579,39 @@ export enum GalarSubRegions {
     CrownTundra,
 }
 
+// Gender Types
+export enum Genders {
+    Genderless = 0,
+    MaleFemale, // Default
+}
+
+// Gender Ratio
+export const FEMALE_0 = 0;
+export const FEMALE_12_5 = 0.125;
+export const FEMALE_25 = 0.25;
+export const FEMALE_50 = 0.5; // Default
+export const FEMALE_75 = 0.75;
+export const FEMALE_87_5 = 0.875;
+export const FEMALE_100 = 1;
+
+// Gender Text
+export const TEXT_GENDERLESS = '';
+export const TEXT_MALE = '<i class="gender-icon-male">♂</i>'; // If Bootstrap Icons: <i class="bi bi-gender-male gender-icon-male"></i>
+export const TEXT_FEMALE = '<i class="gender-icon-female">♀</i>'; // If Bootstrap Icons: <i class="bi bi-gender-female gender-icon-female"></i>
+
+// Wild/Trainer/BF Pokémon Gender
+export enum BattlePokemonGender {
+    NoGender = 0,
+    GenderMale,
+    GenderFemale,
```

Comment:
Is there a reason why they are called "GenderMale" and "GenderFemale"? Gender is already in the enum name, so it should be clear

Reply:
It had another name iirc, changed the names to Male and Female respectively.